MAUI/Picker/populating-items.md

```diff
@@ -105,18 +105,18 @@ The following code example encapsulate how to populate Country and Cityes in eac
 
 ## Set items text style customization
 
-The .NET MAUI Picker control, selected and unselected items, [TextColor], [FontSize], [FontFamily], and [FontAttributes] can be customized as shown in the following code.
+The .NET MAUI Picker control, selected and unselected items, `TextColor`, `FontSize`, `FontFamily`, and `FontAttributes` can be customized as shown in the following code.
```

Comment:
The .NET MAUI Picker control, selected and unselected items, `TextColor`, `FontSize`, `FontFamily`, and `FontAttributes` can be customized as shown in the following code.

Reply:
Content modified.